plugins/flytekit-k8s-pod/flytekitplugins/pod/__init__.py

```diff
@@ -10,4 +14,8 @@
    Pod
 """
 
-from .task import Pod
+warnings.warn(
+    "This pod plugin is no longer necessary, please use the pod_template and pod_template_name options to @task as described "
+    "in https://docs.flyte.org/en/latest/deployment/configuration/general.html#configuring-task-pods-with-k8s-podtemplates",
+    DeprecationWarning,
```

Comment:
```suggestion
    FutureWarning,
```

Deprecation warnings are not displayed by default. For example:

```python
from flytekitplugins.pod import Pod
```

does not print anything with a `DeprecationWarning`.